internal/evaluator/conftest_evaluator_test.go

```diff
@@ -1235,6 +1248,45 @@ func TestRuleMetadata(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "drop stale effectiveOn",
+			result: output.Result{
+				Metadata: map[string]any{
+					"code":        "warning2",
+					"collections": []string{"A"},
```

Comment:
You're too fast! :D After reading it some more I wasn't sure if the `[]any{"A"}` was more realistic compare the real output.

Reply:
Since we have `"collections": []any{"A"}` on the existing test, I think it's probably better.

Sorry for the bad advice.

Reply:
_ninja editor_ :wink: 

I can change it back, if that's better!

Reply:
Yeah, let's be consistent with the other tests.

Reply:
done!